run.py

```diff
@@ -75,26 +80,50 @@ def run(full_text_links, **kwargs):
     else:
         max_queue_size = 0
 
-    logger.info('Publishing records to: CheckIfExtract')
+    if facility_ner and not force_extract:
+
+        logger.info('Publishing records to: task_identify_facilities')
 
-    i = 0
-    total = len(records.payload)
-    for record in records.payload:
-        logger.info(
-            'Publishing [{0:d}/{1:d}]: [{2}]'.format(
-                i+1, total, record['bibcode'])
-        )
+        i = 0
+        total = len(records.payload)
+        for record in records.payload:
+            logger.info(
+                'Publishing [{0:d}/{1:d}]: [{2}]'.format(
+                    i+1, total, record['bibcode'])
+            )
 
-        if max_queue_size and i >= max_queue_size:
-            logger.info('Max_queue_size reached, stopping...')
-            break
+            if max_queue_size and i >= max_queue_size:
+                logger.info('Max_queue_size reached, stopping...')
+                break
+
+            if diagnose:
+                logger.debug("[{}/{}] Calling 'task_identify_facilities' with '{}'".format(i+1, total, str(record)))
+            logger.info("[%i/%i] Calling 'task_identify_facilities' with '%s'", i+1, total, str(record))
+            tasks.task_identify_facilities.delay(record)
```

Comment:
This is the only place where we call task_identify_facilities? Shouldn't it be called also after extraction? Otherwise extraction will not cause the NER to be executed.